server/controller/partyController.js

```diff
@@ -69,10 +77,50 @@ class PartyController{
 
         return res.status(400).send({
             status: 400,
-            message: "Party not found"
+            error: "Party not found"
         })
         
     }
+
+    editAParty(req, res){
```

Comment:
Nice work, You will be making a PATCH request to this endpoint, therefore the values to update the party with will be gotten from the JSON body and not the URL params, Please make the necessary changes for this implementation.

Reply:
ok,i would refactor my code